bower.json

```diff
@@ -10,10 +10,13 @@
     "angular-sanitize": "1.3.0-beta.11",
     "angular-resource": "1.3.0-beta.11",
     "angular-ui-router": "~0.2.10",
+    "bootstrap-sass": "~3.1.0",
     "hello": "~1.3.2",
     "ngCordova": "~0.1.11-alpha"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "platform": "~1.3.0"
```

Comment:
Ahh, is this to emulate different behaviors for different devices? Pretty cool. I see why you made it a dev dependency.